pkg/api/aim/apps.go

```diff
@@ -136,6 +174,7 @@ func DeleteApp(c *fiber.Ctx) error {
 	if err := database.DB.
 		Select("ID").
 		Where("NOT is_archived").
+		Where("namespace_id = ?", ns.ID).
```

Comment:
move this into `app` struct

Reply:
ok, I can do that. We have a mix when we use `where` statements via `model` and when we use `Where` function. I've decided to use `Where` function because we use it more often than inside the `model`.